src/types/JSON.ts

```diff
@@ -0,0 +1,27 @@
+export const SBVRJSON: SBVRType<string, Object> = {
+	types: {
+		postgres: 'TEXT',
+		mysql: 'TEXT',
+		websql: 'TEXT',
+		odata: {
+			name: 'Edm.String' // TODO: What should this really be?
+		},
+	},
+
+ 	fetchProcessing: (data, callback) => {
+		try {
+			callback(null, JSON.parse(data))
+		} catch (e) {
+			callback(e)
+		}
+	},
+
+	validate: (value, required, callback) => {
+		try {
+			callback(null, JSON.stringify(value))
+		} catch (e) {
+			console.error(e)
```

Comment:
Could you add a message to this along with the error - I know the previous version doesn't but it would be good to fix anyway